src/Stratis.Bitcoin/Connection/ConnectionManager.cs

```diff
@@ -164,7 +165,15 @@ public void Start()
                 });
 
                 server.InboundNodeConnectionParameters = cloneParameters;
-                server.Listen();
+                try
+                {
+                    server.Listen();
+                }
+                catch (SocketException e)
+                {
+                    this.logger.LogCritical($"P2P port {listen.Endpoint.Port} is being used. Please use a different port by running the node with '-port=[differentPort]'.");
```

Comment:
https://github.com/stratisproject/StratisBitcoinFullNode/blob/master/Documentation/using-logging.md#interpolated-strings

not critical here as it is one-time exceptional state, but please do for the sake of consistency of the logs

Reply:
Fixed.

Reply:
tx

Reply:
Yes correct I tend to use string interpolation when its not a method that will get hit a lot, but for consistency indeed use string format.